src/components/PoolOverview.tsx

```diff
@@ -75,141 +87,191 @@ export default function PoolOverview({
-                  <span className="label underline">
-                    <Tooltip
-                      title={
-                        <React.Fragment>
-                          {symbol.replaceAll("/", "\n")}
-                        </React.Fragment>
-                      }
+                  <Tooltip title={symbol.replaceAll("/", "\n")}>
+                    <Typography
+                      component="span"
+                      sx={{ borderBottom: "1px dotted" }}
                     >
-                      <React.Fragment>Reward APR</React.Fragment>
-                    </Tooltip>
-                  </span>
+                      Reward APR:&nbsp;
+                    </Typography>
+                  </Tooltip>
                 ) : (
-                  <span className="label">{symbol} APR</span>
+                  <Typography component="span">{symbol} APR: &nbsp;</Typography>
                 )}
-                <span className="plus">
+                <Typography component="span">
                   {formattedData.aprs[key as Partners] as string}
-                </span>
+                </Typography>
               </div>
             ) : null
           })}
-          <div className="margin">
-            <span className="label">TVL</span>
-            <span>{`$${formattedData.reserve}`}</span>
-          </div>
-          {formattedData.volume && (
-            <div>
-              <span className="label">{`${t("24HrVolume")}`}</span>
-              <span>{formattedData.volume}</span>
-            </div>
-          )}
-        </div>
-        <div className="buttons">
-          <Link to={`${poolRoute}/withdraw`} style={{ textDecoration: "none" }}>
-            <Button kind="secondary">{t("withdraw")}</Button>
-          </Link>
-          {shouldMigrate ? (
-            <Button
-              kind="temporary"
-              onClick={onClickMigrate}
-              disabled={!hasShare}
-            >
-              {t("migrate")}
-            </Button>
-          ) : (
-            <Link
-              to={`${poolRoute}/deposit`}
-              style={{ textDecoration: "none" }}
-            >
+        </StyledGrid>
+        <Grid item xs={12} lg={2}>
+          <Stack spacing={2}>
+            {shouldMigrate ? (
               <Button
-                kind="primary"
+                variant="contained"
+                color={isOutdated || shouldMigrate ? "secondary" : "primary"}
+                fullWidth
+                size="large"
+                onClick={onClickMigrate}
+                disabled={!hasShare}
+              >
+                {t("migrate")}
+              </Button>
+            ) : (
+              <Button
+                variant="contained"
+                color={isOutdated || shouldMigrate ? "secondary" : "primary"}
+                fullWidth
+                size="large"
                 disabled={poolData?.isPaused || isOutdated}
+                onClick={() => history.push(`${poolRoute}/deposit`)}
```

Comment:
What is the logic of using a Button with `history.push` rather than just using `Link`?

Reply:
Because we removed css, even though we disable `Button`, it navigate with the help of `Link`. Of course, we can customize link with `styled` but to reduce code I use `hisotry.push` and removed `Link`.